playbooks/02 - Use Case - Asset Change Activity/Medium Impact Baseline Change.json

```diff
@@ -5697,93 +5909,113 @@
             "isExecuted": false,
             "group": null,
-            "uuid": "242d0551-9e31-409f-ae76-27b939b8123a"
+            "uuid": "701641d0-0a90-4d1b-a858-be9fa49af9d6"
         },
         {
             "@type": "WorkflowRoute",
-            "name": "Assignee Update Baseline -> Activity Completed",
-            "targetStep": "\/api\/3\/workflow_steps\/d2f8b90d-6773-4e7d-ac41-2ee63eb20d84",
-            "sourceStep": "\/api\/3\/workflow_steps\/0067b73f-e6e2-4a31-80ac-8c2e8d4f80d8",
+            "name": "Comment Assignee Approval To Implement Task Complete -> Is Approver Assigned",
+            "targetStep": "\/api\/3\/workflow_steps\/a57ed38c-696c-4bc9-8ad5-e1e0f66bced2",
+            "sourceStep": "\/api\/3\/workflow_steps\/68e94c3a-8f0e-4657-9d45-fdc23e2ac78a",
             "label": null,
             "isExecuted": false,
             "group": null,
-            "uuid": "e54c268a-06a4-4dd2-b0d6-7b20c1061993"
+            "uuid": "612d50d0-8082-4c8b-80d4-ae1ba7e6f9c3"
         },
         {
             "@type": "WorkflowRoute",
-            "name": "Assignee Verify Security Controls -> Assignee Update Baseline",
-            "targetStep": "\/api\/3\/workflow_steps\/0067b73f-e6e2-4a31-80ac-8c2e8d4f80d8",
-            "sourceStep": "\/api\/3\/workflow_steps\/97ca00d9-53d4-4e6c-ba61-d30857d46f48",
+            "name": "Comment Identify Security Control Task Complete -> Verify Software Source",
+            "targetStep": "\/api\/3\/workflow_steps\/b4748646-0208-45c2-a1d5-26d7a5e0edc7",
+            "sourceStep": "\/api\/3\/workflow_steps\/e469e875-98a0-4b98-b2b4-5b94fe1efb73",
             "label": null,
             "isExecuted": false,
             "group": null,
-            "uuid": "f26f428b-d379-4e96-bad3-2ea04f3f3a9e"
+            "uuid": "25a897f2-9f92-4848-ba18-03c71cec3781"
         },
         {
             "@type": "WorkflowRoute",
-            "name": "Assignee Verify Software Source -> Assignee Approval To Implement",
+            "name": "Comment Verify Software Source Task Complete -> Assignee Approval To Implement",
             "targetStep": "\/api\/3\/workflow_steps\/94327c0d-46c0-4ce6-ad78-729c4247d9db",
-            "sourceStep": "\/api\/3\/workflow_steps\/b4748646-0208-45c2-a1d5-26d7a5e0edc7",
+            "sourceStep": "\/api\/3\/workflow_steps\/6d3d4d09-af6d-49e3-85e8-aac946167c09",
             "label": null,
             "isExecuted": false,
             "group": null,
-            "uuid": "62d43b6c-88e9-4925-9aae-166401af2cc0"
+            "uuid": "2c7a071e-6a23-41bb-940e-c78ff0f90bcc"
         },
         {
             "@type": "WorkflowRoute",
-            "name": "Assignment And Approver Name -> Do Nothing",
-            "targetStep": "\/api\/3\/workflow_steps\/f6fc3f3d-fa02-4943-9c66-0e50cc903a75",
-            "sourceStep": "\/api\/3\/workflow_steps\/d2aa1894-7f18-4ec6-980d-2bfe0ebd64bf",
-            "label": "Cancel",
+            "name": "Copy of Check Approver 2 -> Implement Change",
```

Comment:
copy?

Reply:
by default it gives copy if any step is copied, BTW - Incorporated